artifactory/cli.go

```diff
@@ -999,6 +999,34 @@ func getRetries(c *cli.Context) (retries int, err error) {
 	return retries, nil
 }
 
+func getRetryWaitTime(c *cli.Context) (waitTime int, err error) {
+	waitTime = cliutils.RetryWaitTime
+	err = nil
+	waitTimeStringValue := c.String("retry-wait-time")
+	useSeconds := false
+	if waitTimeStringValue != "" {
+		if strings.HasSuffix(waitTimeStringValue, "ms") {
+			waitTimeStringValue = strings.TrimSuffix(waitTimeStringValue, "ms")
+		} else if strings.HasSuffix(waitTimeStringValue, "s") {
+			useSeconds = true
+			waitTimeStringValue = strings.TrimSuffix(waitTimeStringValue, "s")
+		} else {
+			err = errors.New("The '--retry-wait-time' option should have a numeric value with 's'/'ms' suffix. " + cliutils.GetDocumentationMessage())
+			return 0, err
+		}
+		waitTime, err = strconv.Atoi(waitTimeStringValue)
+		if err != nil {
+			err = errors.New("The '--retry-wait-time' option should have a numeric value. " + cliutils.GetDocumentationMessage())
```

Comment:
The error message used for the first errro returned in this function is more accurate - 
```
err = errors.New("The '--retry-wait-time' option should have a numeric value with 's'/'ms' suffix. " + cliutils.GetDocumentationMessage())
```
and we can then use it for this case as well. We can create a fucntion that returns this error for reuse.